multisig/src/lib.rs

```diff
@@ -107,8 +107,10 @@ pub trait Multisig:
 
     // ESDT Safe SC calls
 
-    #[endpoint(getNextTransactionBatch)]
-    fn get_next_transaction_batch(&self) -> SCResult<EsdtSafeTxBatchSplitInFields<Self::BigUint>> {
+    #[endpoint(fetchNextTransactionBatch)]
+    fn fetch_next_transaction_batch(
```

Comment:
I do not think we need a multi sig action for fetch_next_transaction_batch - as it should be callable by anyone. There is no consensus to make for that endpoint